src/test/java/it/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMIT.java

```diff
@@ -59,12 +59,12 @@ private BitbucketSCM createSCM(String branchSpec) {
                 new BitbucketSCM(
                         "",
                         ImmutableList.of(new BranchSpec(branchSpec)),
-                        bbJenkinsRule.getCredentialsId(),
+                        bbJenkinsRule.getBitbucketServer().getAdminCredentialsId(),
```

Comment:
This not how this is intended to work (while it will work fine for this test). We have another PR (https://github.com/jenkinsci/atlassian-bitbucket-server-integration-plugin/pull/4) which, once merged, will make this not work any more.

Given you are here could you also fix this timebomb? What we need is a second set of credentials that is of the type `UsernamePasswordCredentialsImpl` and the `adminCredentials` needs to be of an instance of `BitbucketTokenCredentialsImpl`